File: FE-consulting/src/app/Types/GetRateAndReviewDto.ts
export interface GetRateAndReviewDto {
    id: number;
    dateOfVisit: string;
    review: string | null;
    rate: number | null;
    doctorId: string | null;
    patientId: string | null;
    patientName: string | null;
    patientPhoneNumber: string | null;
}

File: Backend-dotnet/GraduationProject.BL/Dtos/Admin/GetRateAndReviewDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GraduationProject.BL
{
    public class GetRateAndReviewDto
    {
        public int Id { get; set; }
        public string DateOfVisit { get; set; }
        public string? Review { get; set; }
        public int? Rate { get; set; }
        public string? DoctorId { get; set; }
        public string? PatientId { get; set; }
        public string? PatientName { get; set; }
        public string? PatientPhoneNumber { get; set; }
    }
}
